gui-selection-list-test-v2-uses-cursor.qml

```diff
@@ -0,0 +1,55 @@
+import QtQuick 2.0
```

Comment:
this and other testiing files don't seem to belong into this PR, but at least and for sure not into the top level directory

Reply:
This was stated in the commit. It will be removed as well as the other commits.
-Dale

Reply:
Oops, reading helps ;-)

Reply:
It was the only way I could get an application shared to test good and bad at once. :)